packages/composite-checkout/src/components/checkout-steps.tsx

```diff
@@ -226,6 +226,7 @@ export function Checkout( {
 
 	const getDefaultCheckoutSteps = () => <DefaultCheckoutSteps />;
 
+	// the `composite-checkout` class name is used by FullStory to avoid recording WordPress.com checkout session activity
```

Comment:
What do you think about adjusting this slightly to make it clear that FullStory is _also_ using this class name but it is not specific to FullStory?

For example:

> Note: the `composite-checkout` class name is also used by FullStory to avoid recording WordPress.com checkout session activity. If this class name is changed or removed, we will also need to adjust this FullStory configuration.